Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,46 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
+The microcontroller and its code are for operating, managing, communicating with, maintaining, and collaborating with all of the other subsystems and acts as both the brain and body of the device.  All the data and instructions will through the microcontroller making it especially important to complete first and giving it a high level of importance.  The hardware will be responsible for the actual actions and completion of any given task.  The code will be responsible for controlling and giving the hardware tasks and manipulating data throughout the system and device.  
+
+Within this subsystem, the code will instruct the hardware in how to sample from the sensors, send data to the Communication subsystem, store data onto a USB Flash drive, and monitor battery voltage levels for the user.  These are the core functions that are desired from the microcontroller.  The microcontroller must be a reasonable portion of the end-user's budget of $300.
+
+## CONSTRAINTS ON THE SUBSYSTEM
```

Comment:
You need to total the number of pins and the type that are required with an explanation. UART takes pins as does other potential operations. then show that the micro has at least than many of the varieties required.